akka-testkit-typed/src/main/scala/akka/typed/testkit/TestInbox.scala

```diff
@@ -27,22 +30,38 @@ class Inbox[T](name: String) {
     new FunctionRef[T](path, (msg, self) ⇒ q.add(msg), (self) ⇒ ())
   }
 
+  /**
+   * Get and remove the oldest message
+   */
   def receiveMsg(): T = q.poll() match {
     case null ⇒ throw new NoSuchElementException(s"polling on an empty inbox: $name")
     case x    ⇒ x
   }
 
+  /**
+   * Assert and remove the the oldest message.
+   */
+  def expectMsg(expectedMessage: String): TestInbox[T] = {
```

Comment:
I don't quite get why it's typed to String? Could be T or Any right?

Reply:
Doh lol